src/Plugin/Field/FieldType/SkosConceptEntityReferenceItem.php

```diff
@@ -17,7 +17,6 @@
  *   id = "skos_concept_entity_reference",
  *   label = @Translation("SKOS Concept Reference"),
  *   description = @Translation("References SKOS Concepts."),
- *   category = @Translation("SKOS"),
```

Comment:
in D11 it was returning an error (https://www.drupal.org/node/3375748). Categories need to be placed in the file MODULE_NAME.field_type_categories.yml). But in D11 when a category is created through the YML file, there is another step in the field form UI. When a category is chosen, the next step is to choose the field type inside the group (even if it's only one field inside the group), so I removed the category to have one step only as it has been so far.